packages/react-components/src/menu/src/Menu.tsx

```diff
@@ -20,66 +23,79 @@ import {
 import { MenuContext } from "./MenuContext";
 import { MenuItem } from "./MenuItem";
 import { MenuSection } from "./MenuSection";
-import { NodeType, useCollection } from "../../collection";
 import { SelectionMode } from "./selectionMode";
```

Comment:
I think SelectionMode could be move back to Menu.tsx. I extracted the enum to prevent circular dependency since Menu import MenuItem and MenuItem was importing SelectionMode from Menu but with `ìmport type` it should be fine.

Reply:
Since SelectionMode is both used as a type (`selectionMode?: SelectionMode;`) and as a value (`selectionMode = SelectionMode.none`), it can not be imported using import type, since it would be removed at runtime.